docs/releases/5.0.txt

```diff
@@ -64,6 +64,10 @@ Minor features
 
 * The default iteration count for the PBKDF2 password hasher is increased from
   600,000 to 720,000.
+* Asynchronous versions of the following functions are now provided, using an
+  ``a`` prefix: :func:`~.contrib.auth.alogin`, :func:`~.contrib.auth.alogout`,
+  :func:`~.contrib.auth.aauthenticate`,
+  :func:`~.contrib.auth.aupdate_session_auth_hash`.
```

Comment:
Blank line above. 

The following asynchronous functions... (they're not async versions of themselves)